examples/basic_infection/README.md

```diff
@@ -0,0 +1,103 @@
+# Infection model: constant force of infection
+The purpose of this example is to model the infection process in a homogeneous population assuming a constant force of infection, such as from the environment.
+
+## Entities, states and variables
+
+- individuals: represent people in a population who can have three states representing their infection status: susceptible, infected, or recovered.
+  - susceptible: individuals with this status can be infected
+  - infected: represents active infection. In this model, infected individuals are not able to transmit to others. After a recovery period, infected individuals update their status to recovered.
+  - recovered: represents recovery from infection. Individuals with this state are not able to get infected.
+- variables:
+  - population size: number of individuals to include in the simulation
+  - force of infection: rate at which susceptible individuals become infected
+  - infection period: time that an individual spends from infection to recovery
+
+## Simulation overview
+The first infection attempt is scheduled at time 0. Infection events are scheduled to occur based on the constant force of infection. Once an infection event is scheduled, a susceptible individual is selected to be infected.  After infection attempt is finished, the next infection event is scheduled based on the constant force of infection. The simulation ends after no more infection events are scheduled.
+
+Infected individuals schedule their recovery at time `t + infected period`. The infection status of recovered individuals remains as recovered for the rest of the simulation.
+
+### Infection state transitions
+```mermaid
+flowchart LR
+S(Susceptible) --FoI--> I(Infected) --inf. period--> R(Recovered)
+```
+## Simulation architecture
+The simulation executaion first initializes the model parameters and prepares the simulation modules.
```

Comment:
Let's discuss in person let's work through moving this from a list structure to something like: at the entry point of the model we read model parameters, initialize each module, set up loop etc. – then what gets kicked off first?Which module schedules infections (transmission manager )? How do they executed? What happens next / which data flows where? 

So the structure is like talking through how things flow through the system so you can put some of the details (like the methods you're describing, stochasticity etc.) in context. Also it's very acceptable to hand wave things you're nto sure about :)